web/pages/index.md

```diff
@@ -19,15 +19,15 @@ hidetitle: True
 <div class="feature-row">
     <!-- Feature 1 -->
     <div class="feature-column">
-        <a class="feature-link" href="https://docs.plasmapy.org/en/latest/notebooks/thomson.html">
+        <a class="feature-link" href="meetings/office_hours">
         <div class="feature-card" 
               style="background-image: linear-gradient(rgba(249, 96, 96, 0.55),
                                        rgba(255, 255, 255, 0.7)), 
                                        url('/images/features/thomson_spectral_density.png')">
```

Comment:
```suggestion
              style="background-image: linear-gradient(rgba(255, 255, 255, 0.5),
                                       rgba(249, 255, 255, 0.5)), 
                                       none;
              background-color: #f96060">
```

This removes the spectral density image from the card and returns it to a solid red that balances with the right most feature card.